draft-irtf-cfrg-opaque.md

```diff
@@ -347,9 +347,7 @@ OPAQUE relies on the following protocols and primitives:
   - Finalize(x, r, Z): Finalize the OPRF evaluation using input `x`,
     random scalar `r`, and evaluation output `Z`, yielding output `y`.
   - DeriveKeyPair(seed): Derive a private and public key pair deterministically from a seed.
-  - SerializedElement: A serialized OPRF group element, a byte array of fixed
-    length.
-  - SerializedScalar: A serialized OPRF scalar, a byte array of fixed length.
+  - Noe: The size of a serialized OPRF group element.
   - Nok: The size of an OPRF private key.
```

Comment:
@kevinlewi Nok is the 'serialized' size, right?

Reply:
👍